source/internal.d.ts

```diff
@@ -47,6 +47,34 @@ export type WordSeparators = '-' | '_' | ' ';
 
 export type StringDigit = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9';
 
+export type Whitespace =
+  | '\u0009' // '\t'
+  | '\u000A' // '\n'
+  | '\u000B' // '\v'
+  | '\u000C' // '\f'
+  | '\u000D' // '\r'
+  | '\u0020' // ' '
+  | '\u0085'
+  | '\u00A0'
+  | '\u1680'
+  | '\u2000'
+  | '\u2001'
+  | '\u2002'
+  | '\u2003'
+  | '\u2004'
+  | '\u2005'
+  | '\u2006'
+  | '\u2007'
+  | '\u2008'
+  | '\u2009'
+  | '\u200A'
+  | '\u2028'
+  | '\u2029'
+  | '\u202F'
+  | '\u205F'
+  | '\u3000'
+  | '\uFEFF';
```

Comment:
Super nitpicky, but I prefer the modern escape syntax: `\u{7A}`.

https://exploringjs.com/es6/ch_unicode.html#sec_escape-sequences